### ISSUE TITLE
Copy surface using correct row pitch

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/Cxbx-Reloaded#1175]
This is a reminder to look into this, once that pull request is merged into master :

https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/commit/4e166b697fd131c822d995d1cbc8f97b21c4d43d#commitcomment-28994989

**Comment by ObiKKa** [comment]
I spotted a rare and recent report on October 15, 2020 of a popup test case message that didn't have an issue here beforehand. I think it may be related to this issue, see further. I'll link the game's issue here.

The message:
"D3DDevice_CopyRects: Failed to copy surface in EmuPatch_D3DDevice_CopyRects"

- [Midnight Club 3: DUB Edition](https://github.com/Cxbx-Reloaded/game-compatibility/issues/916#issuecomment-708697754)
